src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -149,6 +151,8 @@ public void handleHelp() {
 
     void show() {
         primaryStage.show();
+        reminderWindow.show();
+        reminderWindow.fillInnerParts();
```

Comment:
would it be better if the reminderWindow.show() and fillInnerParts() be placed in another method like void showReminder() for better readability?